index.html

```diff
@@ -979,6 +979,11 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
 a cross-origin resource followed by same-origin resources without an explicit
 opt-in, TAO will pass while CORS will fail. Future iterations of this
 specification may attempt to unite those definitions.</p>
+<p class=note>The Timing-Allow-Origin header arrive as part of a cached
```

Comment:
may arrive? Or what do you mean when you say it always arrives as part of a cached response?

Reply:
Added a "may"